src/views/Alteration.vue

```diff
@@ -36,10 +36,10 @@
           Notice which will incur a $100.00 Fee.</p>
       </section>
 
-      <alteration-summary class="mt-10"/>
+      <!-- FUTURE: set `pleaseValidate` when user clicks File and Pay -->
+      <alteration-summary class="mt-10" :pleaseValidate="true" />
```

Comment:
I'm not sure where we want to go with this, but this is one option that should work. (Obviously not using `true` but some value set when user clicks File and Pay.) Or something.